docs/changelog.md

```diff
@@ -20,6 +20,9 @@ permalink: /changelog/
 * **feature-prompts**:
   * Added optional `addressPickerView` and `onManageAddresses` parameters through `AddressDelegate` to `PromptFeature` for a new `AddressPicker` to display a view for selecting addresses to autofill into a site. [#12061](https://github.com/mozilla-mobile/android-components/issues/12061)
 
+* **feature-media**
```

Comment:
One small change needed here: New changelog entries are to be added at the top for this file to later have changes in chronological order.
Will address this in the PR separate PR for landing this.

Reply:
Sure.